$MirOS: ports/shells/zsh/patches/patch-Doc_zsh_texi,v 1.1 2009/09/06 16:31:12 tg Exp $
$OpenBSD: patch-Doc_zsh_texi,v 1.5 2008/11/04 10:00:59 pea Exp $
--- Doc/zsh.texi.orig	Mon Dec 20 15:00:24 2010
+++ Doc/zsh.texi	Tue Mar  8 17:18:07 2011
@@ -7,6 +7,10 @@
 @end iftex
 @setfilename zsh.info
 @settitle zsh
+@dircategory Shells
+@direntry
+* Zsh: (zsh).                  The Z shell.
+@end direntry
 @c %**end of header
 
 @ifinfo
